File: test/fsharpTests/IntTest/toFloatTest.fs
testCase "toFloat(5)" 
<| fun _ -> 
    let expected = 5
    Expect.equal expected (Int.toFloat 5) "error"
testCase "toFloat(0)" 
<| fun _ -> 
    let expected = 0
    Expect.equal expected (Int.toFloat 0) "error"
testCase "toFloat(-7)" 
<| fun _ -> 
    let expected = -7
    Expect.equal expected (Int.toFloat -7) "error"

File: test/ocamlTests/IntTest/toFloatTest.ml
test "toFloat(5)" (fun () -> expect (Int.toFloat 5) |> toEqual Eq.float 5) ; 
test "toFloat(0)" (fun () -> expect (Int.toFloat 0) |> toEqual Eq.float 0) ; 
test "toFloat(-7)" (fun () -> expect (Int.toFloat -7) |> toEqual Eq.float -7) ; 
